components/lists/Signaling.tsx

```diff
@@ -24,6 +24,24 @@ type SearchProp = NativeStackNavigationProp<RootStackParamList, "Search">
 
 const Signaling = forwardRef<SignalingRef, SignalingProps>((_, ref) => {
   const navigation = useNavigation<SearchProp>()
+
+  async function fetchAllFriends() {
+    return api.get("/api/friend-signals").then((response) => response.data)
+  }
+
```

Comment:
Since you are using the arrow function in the below function, can you make this an arrow function as well to keep the consistency? and handle the error as well